### ISSUE TITLE
How to use BERT for ranking with Pairwise loss function

### ISSUE THREAD

**Issue by mdmustafizurrahman** [google-research/bert#761]
Hi I want to use BERT for pairwise loss function. I know how to use it using pointwise loss using the run_classiifer where I can pass query as text_a and passage as a text_b. 

But for pairwise loss, I have a triplet like [query, positive_passage, negative_passage]. So I want BERT to give me like this pos_score = modeling.BertModel(query, positive_passage) and neg_score= modeling.BertModel(query, negative_passage). Then apply Dense layer and SoftMax like in run_classifier.  Note that I want the same modeling.BertModel for both positive and negative passage not two sepearte modeling.BertModel. Please let me know how to do it.

**Comment by BreezeLee** [comment]
Mark. The same question.

**Comment by Allysnow01** [comment]
Mark. The same question.

**Comment by myeclipse** [comment]
is the problem resolve?

**Comment by chaochaoSZ** [comment]
mark

**Comment by chuanfanyoudong** [comment]
mark

**Comment by MyTHWN** [comment]
mark mark
